interactions/models/discord/enums.py

```diff
@@ -611,6 +611,8 @@ class ChannelType(CursedIntEnum):
     """Voice channel for hosting events with an audience"""
     GUILD_FORUM = 15
     """A Forum channel"""
+    MEDIA_CHANNEL = 16
+    """Channel that can only contain threads, similar to `GUILD_FORUM` channels"""
```

Comment:
Not one here to judge, but I don't think us comparing it to threads is semantically accurate. The latter seems to be more agreed on, though, what about this instead?

```suggestion
    """Channel that can only contain media files, similar to `GUILD_FORUM` channels"""
```

Reply:
TYhis is copied verbatim from the dev docs pr